### DISCUSSION TITLE
Transform class: storing too much information in ``data`` may not scale.?

### DISCUSSION THREAD

**Discussion by Jingnan-Jia** [Project-MONAI/MONAI#2763] [Q&A] [answered]
From [here](https://github.com/Project-MONAI/MONAI/blob/4f7371ae4e89f4c1d73652de4144425781a180f0/monai/transforms/transform.py#:~:text=storing%20too%20much%20information%20in%20%60%60data%60%60%20may%20not%20scale.), it shows: 

`storing too much information in ``data`` may not scale.`

I do not know what it mean. Could you please give more explanication on it?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @Jingnan-Jia ,

Thanks for your interest here.
I created PR: https://github.com/Project-MONAI/MONAI/pull/2767 to enhance the doc-string.

Thanks.

**Reply by Jingnan-Jia** [reply]
Do you know how to mark your comment as answer? I can not find the buttom at the up right conor.

**Reply by Nic-Ma** [reply]
Sorry I never marked that.. @wyli @rijobro do you guys know how to mark as answered?

Thanks in advance.

**Reply by wyli** [reply]
That option is available when the category is 'Q&A', I have changed and added this answer
